### PR TITLE
recommiting the changes to ensure a smoother merge

### PR DESCRIPTION
**Pull request by shroffk**
https://github.com/ControlSystemStudio/cs-studio/issues/1855

@berryma4 @kasemir can you accept this pull request

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,68 +1,68 @@
...
+  <groupId>org.csstudio</groupId>
+  <artifactId>maven-osgi-bundles</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <name>maven-osgi-bundles</name>
+
+  <properties>
+    <tycho.version>0.23.1</tycho.version>
+    <manifest-location>META-INF</manifest-location>
+    <cs-studio.version>4.4</cs-studio.version>
```
Comment (berryma4) [inline pom.xml]: in here twice?

File: pom.xml [hunk 2]
```diff
@@ -1,68 +1,68 @@
...
+    <epics.util.version>0.3.2</epics.util.version>
+    <epics.caj.version>1.1.15</epics.caj.version>
+    <epics.jca.version>2.3.6</epics.jca.version>
+    <graphene.version>0.8.0</graphene.version>
+    <pvdata.version>5.0.3</pvdata.version>
+    <pvaccess.version>4.1.3</pvaccess.version>
+    <csstudio.version>4.3.0-SNAPSHOT</csstudio.version>
```
Comment (berryma4) [inline pom.xml]: here again?

**Comment by shroffk** [comment]
I did not add the property...
Will Rodgers had added this property 07fdb1f828f0e0a6207c09b3eae306d6a6483efe

**Comment by willrogers** [comment]
Ha, guilty.

Proably because `cs-studio.version` was used in the other repos, but of course it's incorrect to have both.  I suggest removing `csstudio.version` for consistency.

**Comment by kasemir** [comment]
When updating the epics.caj.version from 1.1.14 to 1.1.15, we also need to update the version of epics/plugins/org.csstudio.platform.libs.epics/pom.xml (and MANIFEST.MF) because it includes the jca and caj libs.
When doing a full rebuild from sources it doesn't matter, because then your local org.csstudio.platform.libs.epics will have the correct jca and caj, but if you fetch   org.csstudio.platform.libs.epics out of thin air based on its version number, the number needs to change when the content changes.

**Comment by shroffk** [comment]
Well, the o.c.p.libs.epics gets it's version number from a property
${csstudio.version}

As I understand our current set up. Building locally as you mentioned should not be a problem, the current master/development p2 repo should be wiped out and replaced with the new one...so people building it out of think air should get the 4.3.x with the new version of the epics libraries.

**Comment by shroffk** [comment]
I do agree that the version number should change to reflect such major internal changes but I am not sure what is the correct thing to do here.

Should we remove the ${csstudio.version} and manually manage this versioning?
Should we use a combination version number, which reflects both the major cs-studio release this library is associated with and the changes within?

**Pull request merged**